src/vivarium_gates_nutrition_optimization/components/hemoglobin.py

```diff
@@ -73,30 +87,10 @@ def setup(self, builder: Builder):
             .set_index(index_columns)["value"]
             .rename("stddev")
         )
-        distribution_parameters = pd.concat([mean, stddev], axis=1).reset_index()
-
-        self.hemorrhage_rr = builder.lookup.build_table(
-            builder.data.load(data_keys.MATERNAL_HEMORRHAGE.RR_ATTRIBUTABLE_TO_HEMOGLOBIN),
-            key_columns=["sex"],
-            parameter_columns=["age", "year"],
-        )
-
-        self.hemorrhage_paf = builder.lookup.build_table(
-            builder.data.load(data_keys.MATERNAL_HEMORRHAGE.PAF_ATTRIBUTABLE_TO_HEMOGLOBIN),
-            key_columns=["sex"],
-            parameter_columns=["age", "year"],
-        )
-
-        self.maternal_disorder_rr = builder.lookup.build_table(
-            builder.data.load(data_keys.MATERNAL_DISORDERS.RR_ATTRIBUTABLE_TO_HEMOGLOBIN),
-            key_columns=["sex"],
-            parameter_columns=["age", "year"],
-        )
-
-        self.maternal_disorder_paf = builder.lookup.build_table(
-            builder.data.load(data_keys.MATERNAL_DISORDERS.PAF_ATTRIBUTABLE_TO_HEMOGLOBIN),
-            key_columns=["sex"],
-            parameter_columns=["age", "year"],
+        distribution_parameters = self.build_lookup_table(
```

Comment:
Nit: shouldn't the code starting at `index_columns` through this `build_lookup_table` call be in this component's `build_all_lookup_tables` method?